conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -917,4 +979,22 @@ public Boolean visitUnknown(String unknownType) {
             }
         });
     }
+
+    private static final class CollectionType {
+        private final Class<?> clazz;
+        private final boolean isPrimitive;
```

Comment:
`isPrimitive` isn't a great name. We're trying to describe the fact that the list class isn't a generic list but a specialized one.

Reply:
I was between that and `isParameterized`. But then started using the same flag for creating an auxiliary setter. So, I renamed it to `isPrimitive`. Maybe there should be two flags. Anyway @carterkozak mentioned leaving this aux. setter optimization for this PR so I'll rename to `isParameterized`

Reply:
[4b0daa1](https://github.com/palantir/conjure-java/pull/2274/commits/4b0daa13de1cf4ff0d2524d957d6712291a07af5)